lib/questions.rb

```diff
@@ -155,6 +185,7 @@ def get_domain_name_from_email_address(email)
 # 'the lion the witch and the wardrobe' becomes
 # 'The Lion the Witch and the Wardrobe'
 def titleize_a_string(string)
+  string.split.map{|word| !['a','and','the'].include?(word) ? word.capitalize : word}[0].capitalize!
```

Comment:
Line is too long. [100/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.